README.md

```diff
@@ -2,6 +2,11 @@
 
 # Bank of Anthos
 
+> **⚠ ATTENTION: Apache Log4j 2 advisory.**  
+> Due to [vulnerabilities](https://cloud.google.com/log4j2-security-advisory) present in earlier versions
+> of Log4j 2, we have taken down all affected container images. We highly recommend all demos and forks to now
+> use images from releases [>= v0.5.2](https://github.com/GoogleCloudPlatform/bank-of-anthos/releases) instead.
```

Comment:
Nit: `instead` is not necessary in this sentence, and is inconsistent with the advisory for microservices-demo.